nomos-libp2p/src/lib.rs

```diff
@@ -56,6 +56,12 @@ pub enum SwarmError {
     DuplicateDialing,
 }
 
+macro_rules! gossipsub_topic {
+    ($str:expr) => {
+        gossipsub::IdentTopic::new($str)
+    };
+}
+
```

Comment:
not sure of the benefit of this

Reply:
Yeah. I wanted to make a `topic_hash(topic: &str)` function to make this code safer: https://github.com/logos-co/nomos-node/pull/292/files#diff-e44f38b567c357a4f950d580460ac8b27205344926ef56acce6a75e3e048801cL130, because there are two types of topic hash representation in libp2p: `IdentityHash` and `Sha256Hash`. Of course, I can make a `topic_hash(topic: &str)` function without having this macro, but wanted to prevent someone from using `gossipsub::Sha256Topic::new(topic)` by mistake.

Reply:
yeah, I don't feel like this one line macro is providing much benefit